### PR TITLE
Decrease BYOS cost to 2500

### PR DESCRIPTION
**Pull request by dee-dubya**
## About The Pull Request
Pretty self explanatory. Changes the cost of the Build your own Shuttle from 5000 to 500.

## Why It's Good For The Game
It's completely ludicrous to have a shuttle that has to be called within the first 10 minutes of the shift cost more funds than the station starts with. However, considering it comes with supplies and has certain benefits, it definitely shouldn't be free or give money either. This corrects the overcorrecting done with the previous BYOS balance change.

## Changelog
:cl: DeeDubya
tweak: Decreased the cost of the BYOS from 5000 to 2500, and removed the time limit for purchasing.
/:cl:

### CODE REVIEW

**Comment by FloranOtten** [comment]
Pretty sure the recent changes made this shuttle able to be called at any time, no?

**Comment by dee-dubya** [comment]
The round start time variable was unchanged in the PR that changed the cost. Not that I think it should be changed, the BYOS should be called early or not at all. Give people time to actually build it, if they so choose.

**Comment by GuyonBroadway** [comment]
For 500 spacebucks you too can grief anyone without access to a spacesuit!

**Comment by Dennok** [comment]
> For 500 spacebucks you too can grief anyone without access to a spacesuit!

Spacesuitless peasants use pods.

And yes. for 10 hits on window / or 15 sec of window disassembling you can grief anyone without access to a spacesuit.

**Comment by GuyonBroadway** [comment]
Indeed but one broken window is easier to fix before you die that a fully airless shuttle.

**Comment by Dennok** [comment]
As i know now it aired.

**Comment by wesoda25** [comment]
Can’t believe the garbage PR making it cost that much was actually merged.

**Comment by zxaber** [comment]
@GuyonBroadway 
Now that the shuttle spawns with walls and atmos, there's a lot less risk of it being a depressurized mess. Which is sad if you ask me, since getting the thing pressurized was like the whole fun of it. 

Unlike other shuttles that got a cost increase, the DIY shuttle wasn't a grief shuttle. There was just literally no reason not to select it, since it gave you (cargo) free money and was a perfectly fine pressurized shuttle. It would honestly be fine at a cost of zero, but anything that's not negative is an improvement.

I really do wish we could just have the old wall-less shuttle again.

**Comment by Dennok** [comment]
> I really do wish we could just have the old wall-less shuttle again.

I'm too.

**Comment by anconfuzedrock** [comment]
building this shuttle is a major effort. it shouldn't be an issue for it to cost as much as your average cargo crate. the shell on its own is an excellent shuttle for several reasons, so making it cheaper is completely idiotic. we don't want captains buying the byos when the crew doesn't want it, and when the crew wants it, they can pay for it.

**Comment by ExcessiveUseOfCobblestone** [comment]
Don't you start with 5K in cargo? It shouldn't be an issue.

**Comment by TheChosenEvilOne** [comment]
@ExcessiveUseOfCobblestone 
Sure when cargo orders insulated gloves, toolbox crate and engineering gear with that.



In my opinion 5000 is way too much for the shuttle and 500 feels a bit too little. I'd say 1000 or 1500 is better (and a less weird feeling price).
Now that I think about it, how much do you actually get if you sold everything inside the shuttle?

**Comment by dee-dubya** [comment]
> Don't you start with 5K in cargo? It shouldn't be an issue.

Station starts with less than 2k from my experience. Assuming someone competent is actually working cargo, there are enough crates and misc garbage in cargo to get over 5k, but that then makes the purchase window extremely slim. If BYOS is going to be an option, it should be an option that the Captain/HoP(or tiding greyshirt) should be able to order when they first check the comms console at the start of a round.

I could bump it up to 1k if people would prefer that, but honestly I don't think that'd make much of a difference. The point here is to make it so that the cost isn't </=0, but is still below 1500.

**Comment by WJohn** [comment]
I thought I had dropped its price to 2500 before packing up but I guess I didn't hit push? Whoops, it wasn't supposed to be 5000.

**Comment by zxaber** [comment]
We could remove the purchase time limit, too. Since it's pretty much good for use when purchased, it doesn't matter if it gets purchased forty-five minutes into the shift right before the shuttle is called.

**Comment by Jordie0608** [comment]
I'd say up the price to at least 1k and increase the timeout.

**Comment by ExcessiveUseOfCobblestone** [comment]
> I thought I had dropped its price to 2500 before packing up but I guess I didn't hit push? Whoops, it wasn't supposed to be 5000.

I'll merge with this change

**Comment by dee-dubya** [comment]
Since more people would rather the time limit get axed and the cost stay a bit higher, I went with that option. I still think the shuttle should be an "early or not at all" option, but it is feasible for a cap or HoP to push a few crates around early on if they're dead set on the shuttle.

File: code/datums/shuttles.dm [hunk 1]
```diff
@@ -180,11 +180,11 @@
...
 	description = "For the enterprising shuttle engineer! The chassis will dock upon purchase, but launch will have to be authorized as usual via shuttle call. Comes stocked with construction materials."
 	admin_notes = "No brig, no medical facilities, no shuttle console."
-	credit_cost = 5000
+	credit_cost = 2500
 
 /datum/map_template/shuttle/emergency/airless/prerequisites_met()
-	// first 10 minutes only
-	return world.time - SSticker.round_start_time < 6000
+	// first 10 minutes only(removed)
+	return world.time - SSticker.round_start_time < 99999999
```
Comment (Jordie0608) [inline code/datums/shuttles.dm]: Why would you do this instead of removing the code block. Even commenting out the block would've been better.

**Pull request merged**